### ISSUE TITLE
Can't configure the plugin.

### ISSUE THREAD

**Issue by mdecaro** [thoukydides/homebridge-homeconnect#298]
### Description of Issue

Can't configure the plugin.  I tried using the UI, also tried manually editing the JSON.  
ERROR LOADING PLUGIN homebridge-homeconnect SyntaxError Unexpected identifier assert

### Expected Behaviour

Get to step where I get QR code for child bridge

### Steps to Reproduce

Used UI or manual install - no luck

### Plugin Version

1.0.3

### Environment

- OS:  macOS 14.6.1
- Node: 22.8.0
- Homebridge:   1.8.3 and 1.8.4

### Home Connect Appliance(s)

Bosch frig

### HomeKit App(s)

Apple Home

### Diagnostic Checks

- [ ] Home Connect app is able to control the appliance with Wi-Fi disabled.
- [ ] Home Connect app shows three green lines in the Network section of the appliance Settings.
- [ ] Unofficial [Home Connect Server Status](https://homeconnect.thouky.co.uk) shows servers up and no recent errors.
- [X] Searched for [previously reported issues](https://github.com/thoukydides/homebridge-homeconnect/issues?q=is%3Aissue) and no similar ones found.
- [ ] Checked the [Home Connect API errors](https://api-docs.home-connect.com/general?#api-errors) and [authorisation errors](https://api-docs.home-connect.com/authorization?#authorization-errors) documentation.

### Log File

```shell
[homebridge-homeconnect] file:///opt/homebrew/lib/node_modules/homebridge-homeconnect/dist/settings.js:3
import PACKAGE from '../package.json' assert { type: 'json' };
                                      ^^^^^^

SyntaxError: Unexpected identifier 'assert'
    at compileSourceTextModule (node:internal/modules/esm/utils:337:16)
    at ModuleLoader.moduleStrategy (node:internal/modules/esm/translators:166:18)
    at callTranslator (node:internal/modules/esm/loader:436:14)
    at ModuleLoader.moduleProvider (node:internal/modules/esm/loader:442:30)

Node.js v22.8.0
```

### Configuration

```json
Empty on new install for that plugin
Tried manually installing with client. - no luck
```

**Comment by thoukydides** [comment]
I suspect that this is due to a change in Node.js 22, probably nodejs/node#51622. It looks like it should be a simple fix to adopt the new syntax for Node.js 22, and that new syntax appears to have been backported to Node.js 18 and 20… but I will need to run some tests to be sure, and will probably need to increase the minimum compatible version numbers on those branches, which will require some users to upgrade.

**Comment by mdecaro** [comment]
In the meantime, should I downgrade to node 20?

**Comment by mdecaro** [comment]
I did downgrade to node 20.17 and it works fine.

**Comment by thoukydides** [comment]
Homebridge officially only supports Node.js LTS releases - which are currently 18 and 20 - so downgrading is a good workaround.

However, Node.js 22 is expected to become an LTS release next month (October 2024) so I do need to fix this.

It looks like the fix will require a minimum of Node.js 18.20.0.
Node.js 19 (which is not an LTS release) will **not** be supported.

**Comment by thoukydides** [comment]
Should be fixed in [v1.0.4](https://github.com/thoukydides/homebridge-homeconnect/releases/tag/v1.0.4).
